lib/iris/fileformats/pp_rules.py

```diff
@@ -766,6 +777,7 @@ def convert(f):
     aux_coords_and_dims = []
 
     # "Normal" (non-cross-sectional) Time values (--> scalar coordinates)
+    # Our spurious scalar time coord gets introduced by this call...
```

Comment:
Nooo, it wasn't, and I thought I'd removed it! It'll be gone by the next commit.